Net07.HW/Net07.WorkWithDate/Net07.WorkWithDate/Program.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+
+namespace Net07.WorkWithDate
+{
+    class Program
+    {
+        enum DaysInMonths
+        {
+            jan = 31,
+            lFeb = 29,
+            Feb = 28,
+            mar = 31,
+            apr = 30,
+            may = 31,
+            jun = 30,
+            jul = 31,
+            aug = 31,
+            sep = 30,
+            oct = 31,
+            nov = 30,
+            dec = 31
+        }
+        enum DaysOfWeek
```

Comment:
Please, keep empty lines between enums and methods

Reply:
fixed